index.js

```diff
@@ -56,13 +56,19 @@ function flatten(arrays) {
 
 
 function getSimulatorInfo(extensionDir) {
+  var binaryDir = currentPlatform.simulatorBinary;
   var simulatorRegex = /fxos_(.*)_simulator@mozilla\.org$/;
   var matches = simulatorRegex.exec(extensionDir);
   if (matches && matches[1]) {
     var version = matches[1].replace('_', '.');
+
+    if (process.platform === 'darwin') {
+      binaryDir = binaryDir['version' + matches[1].replace('_', 'x')] || binaryDir.standardPath;
```

Comment:
So the binary has `x` in the path instead of underscores everywhere? Sorry, just a bit confused in terms of the order of things happening here.